packages/headless/src/utils/highlight.ts

```diff
@@ -59,18 +62,18 @@ export interface SuggestionHighlightingOptions {
   correctionDelimiters?: Delimiters;
 }
 
+function haveEmptyDelimiters(delimiters: Delimiters) {
+  return delimiters.close === '' || delimiters.open === '';
+}
+
 function isEmptyString(str: string) {
   return str === '';
 }
-
 /**
  * Highlight the passed string using specified highlights.
  */
 export function highlightString(params: HighlightParams): string {
-  if (
-    isEmptyString(params.openingDelimiter) ||
-    isEmptyString(params.closingDelimiter)
-  ) {
+  if (haveEmptyDelimiters(params.delimiters)) {
     throw Error('delimiters should be a non-empty string');
```

Comment:
Is that error handled somewhere?

Reply:
No. I should change `atomic-result-value` to catch this. 

But I tested with an empty string and the result shown on the page was `undefined`. I'm thinking, instead of throwing, we could use pino to log the error and return the result so that the page does not show `undefined`.  Thoughts?